com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -35,8 +34,12 @@ public override VisualElement CreateInspectorGUI()
             var volumeProp = serializedObject.FindProperty(() => Target.BoundingShape2D);
             ux.Add(new PropertyField(volumeProp));
             ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.Damping)));
+            var oversizedCameraHelp = ux.AddChild(new HelpBox(
+                "The camera window size is bigger than what can fit perfectly in the confiner. Enable Oversize Window option.",
```

Comment:
Awkward wording.  This is better:
"The camera window too big for the confiner. Enable the Oversize Window option."